src/api/filter-resources-data.ts

```diff
@@ -0,0 +1,32 @@
+function filterResourcesData(input, data) {
+  if (input === '') {
+    return data;
+  }
+
+  // Not a regex expert but this seemed cleaner than casting everything to lower case
+  const caseInsensitiveRegex = new RegExp(input, 'i');
```

Comment:
```suggestion
  const caseInsensitiveInput = new RegExp(input, 'i');
```

Adding a suggestion for later (GitHub is having issues so I'm having trouble pushing changes)